### PR TITLE
Move cloud login forgot password link into card actions area

### PR DESCRIPTION
**Pull request by codyc1515**
## Proposed change
Aligns login forgot password link styling with create resend confirmation styling.

### Example
Web capture_3-2-2024_15542_homeassistant local
Web capture_3-2-2024_155435_homeassistant local

## Type of change
- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information
- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/cloud/login/cloud-login.ts [hunk 1]
```diff
@@ -163,6 +154,15 @@ export class CloudLogin extends LitElement {
                   )}</ha-progress-button
                 >
               </div>
+              <button
```
Comment (bramkragten) [inline src/panels/config/cloud/login/cloud-login.ts:157]: This is now outside of `.card-actions`, have you tested this PR?

**Comment by bramkragten** [comment]
Your current code looks like this:

CleanShot 2024-02-05 at 13 48 50

Even if you placed the element inside the correct div, it would look like this:
CleanShot 2024-02-05 at 13 49 42

Please test your PR's before creating them or we will not accept them.

File: src/panels/config/cloud/login/cloud-login.ts (hunk 1 above)
Comment (codyc1515) [reply src/panels/config/cloud/login/cloud-login.ts]: No I hadn't and didn't check that box when submitting the PR. I now have a proper dev environment set-up after many hours of trial & error and delighted to say that I have this working now, which is amazing! Sorry if you felt I wasted your time here with this, it was not my intent.

**Review by bramkragten — APPROVED** [review]

**Comment by codyc1515** [comment]
Tested working on 2024.2.1

**Pull request merged**